src/Servers/IIS/IIS/src/StartupHook.cs

```diff
@@ -83,7 +83,10 @@ public static void Initialize()
                 new PhysicalFileProvider(contentRoot),
                 sourceCodeLineCount: 6);
 
+            // The startup hook is only present when detailed errors are allowed, so
+            // we can turn on all the details.
             model.ErrorDetails = exceptionDetailProvider.GetDetails(exception);
+            model.ShowRuntimeDetails = true;
```

Comment:
This is necessary since the property defaults to `false`. I thought about defaulting it to `true` but I think it's better for users of the page to opt-in to detailed information.

Reply:
There may be one more place we do this. IIRC we have a hosting startup error page and a detailed errors error page. Can you check these and make sure we set ShowRuntimeDetails appropriately:
- https://github.com/aspnet/AspNetCore/blob/cc1f23c5f8afb7d2a00405f19811d2372c4fcec2/src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddleware.cs
- https://github.com/aspnet/AspNetCore/blob/f3f9a1cdbcd06b298035b523732b9f45b1408461/src/Hosting/Hosting/src/GenericHost/GenericWebHostedService.cs
- https://github.com/aspnet/AspNetCore/blob/f3f9a1cdbcd06b298035b523732b9f45b1408461/src/Hosting/Hosting/src/Internal/WebHost.cs

Reply:
I checked the diagnostics one. It's a little deceptive because it's actually a different error page because it's even more detailed (query strings, headers, etc.) and the middleware is only enabled in `Development` mode.

I did miss GenericWebHostedService. Looking at that now.

Reply:
Actually it was the other way around :). I missed the WebHost one. Updated.

Reply:
Sorry, clarify why you don't need to updat ethe GenericWebHostedService model?

Reply:
Because I already did: https://github.com/aspnet/AspNetCore/pull/12545/files#diff-712c8e185558d5014acb35f2f2b86786